tests/test_hll.py

```diff
@@ -240,9 +240,6 @@ def test_hll_change_error_rate():
     with pytest.raises(TypeError):
         del hllcpp.error_rate
 
-    with pytest.raises(TypeError):
```

Comment:
Removed this test because it made sense for the original code, but Cython will convert integers to floats (and then print an error, because the error rate need in a range in (0, 1) but not containing any of them).